packages/itwinui-css/src/tag/tag.scss

```diff
@@ -59,6 +59,7 @@
 
 @mixin iui-tag-container {
   color: var(--iui-text-color-muted);
+  column-gap: $iui-xs;
```

Comment:
Don't you need to remove?
```sass
> *:not(last-child) {
  margin-right: $iui-xs
}
```

Reply:
Actually, because `iui-tag-container` does not have `display: flex`, I cannot use `gap` here